content/docs/atom-guide/look-dev/tutorials.md

```diff
@@ -0,0 +1,14 @@
+---
+linktitle: Tutorials
+title: "Tutorials"
+description: "Tutorials for the Atom renderer."
+toc: true
+---
+
+To learn how to use the Atom renderer, refer to any of these [rendering tutorials](/docs/learning-guide/tutorials/rendering/)!
+
+| Tutorial | Description |
+| - | - |
+| [Create a StandardPBR material](/docs/learning-guide/tutorials/rendering/create-standardpbr-material/) | This tutorial walks you through creating your first StandardPBR material using the Material Editor. | 
+| [Material Types and Shaders](/docs/learning-guide/tutorials/rendering/get-started-materialtypes-and-shaders/) | A beginner-level tutorial introducing material types and shaders in Atom. Learn how to create a custom material type with a simple AZSL shader. | 
+| [Vegetation Bending Custom Material Type](/docs/learning-guide/tutorials/rendering/vegetation-bending-tutorial) | Use the **Atom Renderer** to create a custom material type with custom vertex shaders for vegetation bending. |
```

Comment:
Nit: let's rename 'Vegetation Bending Custom Material Type' to 'Vertex Deformation for Vegetation Bending'. I don't think we need to draw attention to the fact that it's done via a custom material type, but I do think it's important to draw attention to the fact that this isn't really a tutorial about vegetation bending, it's a tutorial about how to do more general vertex deformation using vegetation bending as an example.

Reply:
This is a good suggestion, renamed!